weave/trace_server/refs_internal.py

```diff
@@ -10,6 +10,7 @@
 
 WEAVE_INTERNAL_SCHEME = "weave-trace-internal"
 WEAVE_SCHEME = "weave"
+PRIVATE_WEAVE_STR = "<private>"
```

Comment:
I would use something more distinct, for example `<<PRIVATE_REF_DATA>>`

Reply:
i wonder if we should do something like:
`weave-private://///{rest}` (yes, 5 slashes).

Reply:
that would still meet our criteria of not being an official "ref", but also retain the "rest" component (name, version, etc...) if we did want to parse it in the future

Reply:
yeah I like it